
map_image_save_dir: /home/yds/map.png
map_config_save_dir: /home/yds/map.yaml

map:
  sizex: 650
  sizey: 650
  initx: 0
  inity: 0
  cell_size: 0.2
  frame_id: odom
  
robot_laser:
  x: 0.0
  y: 2.34
  theta: 1.5708

sensor_model:
  P_occ: 0.8
  P_free: 0.2
  P_prior: 0.5
  
robot:
  kl: 2.31249273072714e-06
  kr: 2.30439760302837e-06
  b: 0.719561485930414
